### DISCUSSION TITLE
Sounds for Oi, Grandad!

### DISCUSSION THREAD

**Discussion by RickTaylor79** [publicsamples/Oi-Grandad#6] [Ideas]
I've found the following freely available sounds to work nicely with Oi, Grandad! Please feel free to share any you come across.

These files created for the Make Noise Morphagene are really special and work well with OG:

https://freesound.org/people/makenoisemusic/sounds/?page=1#sound

Old 78 RPMs and Cylinder Recordings:

https://archive.org/details/78rpm

Yours truly:

[Older OG preset pack](https://github.com/publicsamples/Oi-Grandad/releases/download/1.1.6/Samples.Presets.zip)
[Metasynth Loops](https://github.com/publicsamples/Metasynth-Loops) 
[Modular-Loops](https://github.com/publicsamples/Modular-Loops/releases/tag/1.0) 
[Former Loops](https://github.com/publicsamples/Former-Loops-/releases/tag/1.0)

And if you don't mind spending some change:
[
Oi, Grandad!: Ultra Pack Pro](https://modularsamples.gumroad.com/l/gyckl)
[Untitled Loop Project](https://modularsamples.gumroad.com/l/TdDvx)

**Comment by alexis-haklaj** [comment]
Hi! Can you elucidate me about which items need to be downloaded to use "Modular-Loops" and "Former Loops", to  how and where  to place them to use it on ableton, windows?

**Reply by RickTaylor79** [reply]
Hello! 

In both cases, you need to download the two mutli-part zip files (001 & 002) and extract them with [7zip](https://www.7-zip.org).  After that where you place them is up to you, but the app data folder for Oi, Grandad! is generally where I'd recommend. It helps keep everything in one place:

` C:\Users\AppData/Roaming\modularsamples`

To load the samples in v1.9 simply drag and drop, or right click to bring up a file browser. If you're using 2.2, have a look at this wiki page:

https://github.com/publicsamples/Oi-Grandad/wiki/File-management-in-Oi,-Grandad!-v2.2b

**Comment by alexis-haklaj** [comment]
Uau! Amazing! thanks, and really, great job with all the stuff you create
and find! keep!

Em seg., 30 de jun. de 2025 às 15:49, Rick ***@***.***>
escreveu:
